### PR TITLE
requirements.txt: Selenium v4.27.0 fixes TypeError on Safari init

### PR DESCRIPTION
**Pull request by cclauss**
Reverts the Selenium change in:
* #5182

Selenium v4.27.0 fixes TypeError when init Safari webdriver
* https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.27.0

### Checklists

- [ ] Add a [CHANGELOG](https://github.com/pyodide/pyodide/blob/main/docs/project/changelog.md) entry
- [ ] Add / update tests
- [ ] Add new / update outdated documentation

### CODE REVIEW

**Review by agriyakhetarpal — APPROVED** [review]
Thanks, @cclauss!

File: requirements.txt [hunk 1]
```diff
@@ -6,13 +6,10 @@ build~=1.2.0
 sphinx-click
 hypothesis
 mypy==1.11.0
-# (FIXME: 2024/01/28) The latest pytest-asyncio 0.23.3 is not compatible with pytest 8.0.0
-pytest<8.0.0
+pytest
```
Comment (cclauss) [inline requirements.txt:9]:
```suggestion
# (FIXME: 2024/01/28) The latest pytest-asyncio 0.23.3 is not compatible with pytest 8.0.0
pytest<8.0.0
```

**Comment by cclauss** [comment]
Please rerun the failing CircleCI test.

**Comment by agriyakhetarpal** [comment]
Thanks, I've just re-triggered it – it does seem like LightGBM is a consistent failure over the past few days, though.

**Comment by cclauss** [comment]
Rebasing made all CI tests pass.  😄

**Pull request merged**